vllm/platforms/interface.py

```diff
@@ -34,6 +34,7 @@ class _Backend(enum.Enum):
     TORCH_SDPA = enum.auto()
     OPENVINO = enum.auto()
     FLASHINFER = enum.auto()
+    FLASHINFER_VLLM_V1 = enum.auto()
```

Comment:
I don't think there is much of a difference here. you can still use previous enums for this